test/functional/configuration/configuration.spec.js

```diff
@@ -118,7 +129,8 @@ test("initializes form fields with full settings", async t => {
             urlDestinationsEnabled: true,
             cookieDestinationsEnabled: false,
             prehidingStyle: "#container { display: none }",
-            context: ["device", "placeContext"]
+            context: ["device", "placeContext"],
+            clickCollectionEnabled: false
```

Comment:
It would probably make sense to add `downloadLinkQualifier` with some custom regex to the second instance object, just so you can assert that the field gets populated with a custom value.